.travis.yml

```diff
@@ -6,7 +6,6 @@ node_js:
 install:
   - npm install
   - npm run build
-  - npm run build:respec-w3c-common
```

Comment:
We should add `npm run parcel` here, but `npm run parcel` never ends on my local environment. Not sure what's happening.